assests/javascript/index.js

```diff
@@ -1 +1,225 @@
-//
+const apiKey = "8cf248a4fd398e2908cf52a9f187375b";
+const mainSection = document.getElementById("main");
+
+let city = {};
+let curentWeather = {};
+let dailyForecastingWeatherList = {};
+
+getWeather();
+
+async function getWeather() {
```

Comment:
wrap the logic of this function in try and catch. In catch you can show the error message.

Reply:
that was greaaaat idea thanksssss